lightweightsemaphore.h

```diff
@@ -26,6 +26,8 @@ extern "C" {
 #include <mach/mach.h>
 #elif defined(__unix__)
 #include <semaphore.h>
+#elif defined(__MVS__)
```

Comment:
This makes the glibc version detection below only apply on MVS

Reply:
Oh, sorry, I'm not really good at programming 😅 
How would you suggest I fix this, move it above `#elif defined(__unix__)` ?

Reply:
Yes, I think that makes sense.

Reply:
Okay, I'll do this later today and push, thank you!